vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutIT.java

```diff
@@ -102,23 +79,19 @@ public void dialogOpenedWithListenOnShortcut_dialogReopened_oldShortcutStillWork
         pressShortcutKey(getFirstDialogInput());
         validateLatestShortcutEventOnDialog(1, 0);
 
-        pressShortcutKey(uiLevelButton);
-        validateLatestShortcutEvent(2, DialogWithShortcutPage.UI_BUTTON);
```

Comment:
Here the test also opens a modal dialog and then presses a shortcut after focusing an element in the background. Which doesn't really make sense in how the should UI work, and is also prevented by server-side modality.